ang/crmCaseType/edit.html

```diff
@@ -25,10 +25,22 @@ <h1 crm-page-title>{{caseType.title || ts('New Case Type')}}</h1>
       <li><a href="#acttab-statuses">{{ts('Case Statuses')}}</a></li>
       <li><a href="#acttab-actType">{{ts('Activity Types')}}</a></li>
       <li ng-repeat="activitySet in caseType.definition.activitySets">
-        <a href="#acttab-{{$index}}">{{ activitySet.label }}</a>
+        <a href="#acttab-{{$index}}">
+          <span ng-if="!activitySet.isInEditMode">{{ activitySet.label }}</span>
+          <input ng-if="activitySet.isInEditMode"
+            class="crmCaseType-edit-actset"
+            ng-keydown="$event.stopImmediatePropagation()"
+            ng-model="activitySet.label" />
+        </a>
+        <span ng-if="!activitySet.isInEditMode" class="crm-i fa-pencil" title="{{ts('Edit')}}"
+          ng-click="activitySet.isInEditMode = true"></span>
+
+        <span ng-if="activitySet.isInEditMode" class="crm-i fa-save" title="{{ts('Remove')}}"
+          ng-click="activitySet.isInEditMode = false"></span>
+
         <span class="crm-i fa-trash" title="{{ts('Remove')}}"
           ng-hide="activitySet.name == 'standard_timeline'"
-          ng-click="removeItem(caseType.definition.activitySets, activitySet)">{{ts('Remove')}}</span>
+          ng-click="removeItem(caseType.definition.activitySets, activitySet)"></span>
```

Comment:
Why did you remove this text? I know it's not visible anyways, but maybe it was there for accessibility reasons (not 100% sure).

Reply:
It was visible before, but I removed it to save space. It is also mentioned in the DOC
`Removed the "Remove" label from the trash icon, because otherwise it was taking lot of space.`

Reply:
Ah right sorry :man_facepalming:

Reply:
For accessibility, I suggest adding back a title "Remove Activity Set"